web/components/input/buttons/CheckboxGroup/index.js

```diff
@@ -10,19 +10,19 @@ const Checkbox = ({ name, value, label, className, checked, disabled, onChange }
       (context) => {
         const contextOnChange = event => context.onChange(event.target.value);
         return (
-          <div className={`checkbox user-select-none margin-bottom-s margin-left-s ${className}`}>
-            <label className="clickable">
+          <div className={`checkbox user-select-none margin-bottom-s margin-right-s ${className}`}>
```

Comment:
Actually can we try something like this? And with a greater margin value (I think in the design it's 56px of margin between elements)
https://stackoverflow.com/questions/8184958/add-space-between-html-elements-only-using-css

Reply:
Sorry I'm not sure which margin you're referring to, do you mean the horizontal spacing between each checkbox/label pair or the vertical spacing? I think Stanford fixed the horizontal spacing in a PR